src/content/docs/hi/basics/project-structure.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: परियोजना संरचना
+description: Learn how to structure a project with Astro.
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+
+`create astro` CLI विज़ार्ड से उत्पन्न आपके नए Astro परियोजना में पहले से ही कुछ फ़ाइलें और फ़ोल्डर्स शामिल होते हैं। अन्य, आप स्वयं बनाएंगे और Astro की मौजूदा फ़ाइल संरचना में जोड़ देंगे।
+
+यहां बताया गया है कि Astro परियोजना कैसे आयोजित किया जाता है, और कुछ फ़ाइलें आपको अपने नए परियोजना में मिलेंगी।
+
+## निर्देशिकाएँ और फ़ाइलें
+
+Astro आपके परियोजना के लिए एक सुविचारित फ़ोल्डर लेआउट का लाभ उठाता है। प्रत्येक Astro परियोजना रूट में निम्नलिखित निर्देशिकाएँ और फ़ाइलें शामिल होनी चाहिए:
+
+- `src/*` - आपका परियोजना स्रोत कोड (अवयव, पृष्ठ, शैलियाँ, आदि।)
+- `public/*` - आपका गैर-कोड, असंसाधित संपत्ति (फ़ॉन्ट, आइकॉन, आदि)
+- `package.json` - एक परियोजना घोषणापत्र।
+- `astro.config.mjs` - एक Astro कॉन्फ़िगरेशन फ़ाइल। (अनुशंसित)
+- `tsconfig.json` - एक TypeScript कॉन्फ़िगरेशन फ़ाइल। (अनुशंसित)
+
+### उदाहरण परियोजना वृक्ष
+
+एक सामान्य Astro परियोजना निर्देशिका इस तरह दिख सकती है:
+
+<FileTree>
+- public/
+  - robots.txt
+  - favicon.svg
+  - social-image.png
+- src/
+  - components/
+    - Header.astro
+    - Button.jsx
+  - layouts/
+    - PostLayout.astro
+  - pages/
+    - posts/
+      - post1.md
+      - post2.md
+      - post3.md
+    - index.astro
+  - styles/
+    - global.css
+- astro.config.mjs
+- package.json
+- tsconfig.json
+</FileTree>
+
+### `src/`
+
+`src/` फ़ोल्डर वह जगह है जहां आपका अधिकांश परियोजना स्रोत कोड रहता है। इसमें शामिल:
+
+- [पेजों](/hi/basics/astro-pages/)
+- [लेआउट](/hi/basics/layouts/)
+- [Astro अवयव](/hi/basics/astro-components/)
+- [UI फ्रेमवर्क अवयव (React, etc.)](/hi/guides/framework-components/)
+- [शैलियां (CSS, Sass)](/hi/guides/styling/)
+- [Markdown](/hi/guides/markdown-content/)
+
+ब्राउज़र पर भेजी जाने वाली अंतिम वेबसाइट बनाने के लिए Astro आपकी `src/` फ़ाइलों को संसाधित, अनुकूलित और बंडल करता है। स्थिर `public/` निर्देशिका के विपरीत, आपकी `src/` फ़ाइलें Astro द्वारा आपके लिए बनाई और प्रबंधित की जाती हैं।
+
+कुछ फ़ाइलें (जैसे Astro अवयव) लिखित रूप में ब्राउज़र पर भी नहीं भेजी जाती हैं, बल्कि उन्हें स्थिर HTML में प्रस्तुत किया जाता है। अन्य फ़ाइलें (जैसे CSS) ब्राउज़र पर भेजी जाती हैं लेकिन प्रदर्शन के लिए उन्हें अन्य CSS फ़ाइलों के साथ अनुकूलित या बंडल किया जा सकता है।
+
+:::tip
+हालाँकि यह मार्गदर्शिका Astro समुदाय में उपयोग की जाने वाली कुछ लोकप्रिय परंपराओं का वर्णन करती है, Astro द्वारा आरक्षित एकमात्र निर्देशिकाएँ `src/pages/` और `src/content/` हैं। आप किसी भी अन्य निर्देशिका का नाम बदलने और उसे इस तरह से पुनर्व्यवस्थित करने के लिए स्वतंत्र हैं जो आपके लिए सर्वोत्तम हो।
+:::
+
+### `src/assets`
```

Comment:
I didn't found this is [en/basics/project-structure](https://docs.astro.build/en/basics/project-structure/#src). Is there a reason why this is included ? Open question to other languages as well

Reply:
your right, what I did was take a glance in [es/basics/project-structure](https://docs.astro.build/es/basics/project-structure/#srcassets), took an assumption that it's a "special information content". I will remove `assets` 👍